### ISSUE TITLE
Date s

### ISSUE THREAD

**Issue by jttaggart** [Roche/pyreadstat#170]
Hi,

Pyreasdstat 1.1.4
Windows 10
SPSS 26.0

I've noticed something odd when working with dates. Dates are written as string types. mm/dd/yyyy has value as number but string type. dd-mmm-yyyy has value as yyyy-mm-dd and string type  (as opposed to yyyy-MON-dd).

See below:

Where  Name is the SPSS format.

Name: mm_dd_yyyy original_type: ADATE10 Type <class 'str'>
=====
Value: 13797216000.0
Value: 13799980800.0

Name: hh_mm_ss original_type: TIME8 Type <class 'str'>
=====
Value: 12:01:01
Value: 03:00:00

Name: dd_mmm_yyyy original_type: DATE11 Type <class 'str'>
=====
Value: 2020-01-01
Value: 2020-02-02

Name: mm_dd_yy original_type: ADATE8 Type <class 'str'>
=====
Value: 13890873600.0
Value: 13860547200.0

Name: dd_mm_yyyy original_type: EDATE10 Type <class 'str'>
=====
Value: 2022-12-30
Value: 2022-03-01

Attached file has: date_time.py, date_time.sav

Regards.

[date_time.zip](https://github.com/Roche/pyreadstat/files/8256348/date_time.zip)

**Comment by ofajardo** [comment]
Hi, thanks for the reproducible report. 

SPSS continuously adds new Date formats, and I have to add them manually to the code. I just did for the ones you reported and a couple more I have found in the SPSS [documentation](https://libguides.library.kent.edu/SPSS/DatesTime). The problem is now solved in the dev branch and will become available in the next release. 

Just as you seem to have a bit of confusion: in SPSS dates, datetimes and times are always stored as a number, what you see in SPSS is just a format. In python we translate those numbers to a datetime.date or datetime.time object, not to a string, when you do this in your script: 

```
print("Name:",  var, "original_type:", meta.original_variable_types[var], "Type", type(var))
```

you are printing the type of the name of the column stored in meta, which of course is a string, but if you would do: 

```
for var_value in df_spss[var]:
		print("Value:", var_value)
                 print("Type:", type(var_value)
```

then you would see it is a python datetime.date, datetime.time or datetime.datetime
